fedimint-rocksdb/src/lib.rs

```diff
@@ -51,6 +51,27 @@ impl From<RocksDb> for rocksdb::OptimisticTransactionDB {
     }
 }
 
+// When finding by prefix iterating in Reverse order, we need to start from
+// "prefix+1" instead of "prefix", using lexicographic ordering. See the tests
+// below
+fn next_prefix(prefix: &[u8]) -> Result<Vec<u8>> {
+    let mut next_prefix = prefix.to_vec();
+    let mut increased = false;
+    for i in (0..next_prefix.len()).rev() {
+        if next_prefix[i] < 0xff {
```

Comment:
`if next_prefix[i] == 0xff` seems easier to grasp.

Reply:
That `increased` is kind of meh name. We always going to increase that number ... kind of ... . Maybe `non_zero_digit`? Especially with my version it seems to be a bit more clear.

Reply:
Take a look now